File: finah-desktop-java/src/net/finah/API/Vraag.java
package net.finah.API;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Vraag {
	@JsonProperty("Id")
	private int id;

	@JsonProperty("Beschrijving")
	private String beschrijving;

	@JsonProperty("Vragenlijst_Id")
	private int vragenlijst_Id;

	public String toString(){
		return id + ": " + beschrijving + ":" + vragenlijst_Id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public void setBeschrijving(String beschrijving) {
		this.beschrijving = beschrijving;
	}

	public void setVragenLijst_Id(int vragenlijst_Id) {
		this.vragenlijst_Id = vragenlijst_Id;
	}
	
}

File: net.finah/API/Vraag.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace API
{
    class Vraag
    {
        private int id;
        private string beschrijving;
        private int vragenlijstId;

       /* public Vraag(int id, string beschrijving, int vragenlijstId) 
        {
            this.id = id;
            this.beschrijving = beschrijving;
            this.vragenlijstId = vragenlijstId;
        }*/

        public int setId
        {
            get { return id; }
            set { id = value; }
        }

        public string setBeschrijving
        {
            get { return beschrijving; }
            set { beschrijving = value; }
        }

        public int setvragenlijstId
        {
            get { return vragenlijstId; }
            set { vragenlijstId = value; }
        }

        public String toString()
        {
            return id + ": " + beschrijving;
        }
      
    }
}